consumers/hidden-test-consumer/src/hidden_test_consumer/hidden_test_process_pb2_grpc.py

```
@@ -0,0 +1,97 @@
+_version_not_supported = False
+
+try:
+    from grpc._utilities import first_version_is_lower
+    _version_not_supported = first_version_is_lower(GRPC_VERSION, GRPC_GENERATED_VERSION)
+except ImportError:
+    _version_not_supported = True
+
+if _version_not_supported:
+    raise RuntimeError(
+        f'The grpc package installed is at version {GRPC_VERSION},'
+        + f' but the generated code in hidden_test_process_pb2_grpc.py depends on'
```

Ruff (0.8.2)

Explicitly concatenated string should be implicitly concatenated

(ISC003)